web_environment_ribbon/static/src/js/ribbon.js

```diff
@@ -1,35 +1,57 @@
-/******************************************************************************
-    Copyright (C) 2015 Akretion (http://www.akretion.com)
-    @author Sylvain Calador <sylvain.calador@akretion.com>
+/* Copyright 2015 Sylvain Calador <sylvain.calador@akretion.com>
+   Copyright 2015 Javi Melendez <javi.melendez@algios.com>
+   Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
 
-    This program is free software: you can redistribute it and/or modify
-    it under the terms of the GNU Affero General Public License as
-    published by the Free Software Foundation, either version 3 of the
-    License, or (at your option) any later version.
+(function ($) {
```

Comment:
I've followed this guidelines (https://github.com/OCA/maintainer-tools/blob/ca0941c654a4c319686aff9cd82b66ab0fb9a06b/template/module/static/src/js/module_name.js#L4) to migrate to v9.
I didn't know any new JS API in v9, could you indicate where to find it?